firmware/application/external/external.ld

```diff
@@ -0,0 +1,37 @@
+/*
+    Copyright (C) 2023 Bernd Herzog
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+MEMORY
+{
+    /* external apps: regions can't overlap so addresses are corrected after build */
+    ram_external_app_pacman     (rwx) : org = 0xEEE90000, len = 40k
```

Comment:
Interesting -- so to add an app, you have to modify this too? I wonder if CMake could generate this based on the apps defined.

Reply:
It should be possible to generate this. Maybe we should do this later. But manually writing it leaves more flexibility with the memory management.